docs/setup-cluster/checklists/_index.rst

```diff
@@ -315,11 +315,47 @@ Non-Root Containers
 ===================
 
 You can enhance security and limit potential malicious activity by running containers as non-root
-users. To find out more, visit :ref:`run-as-user` and :ref:`run-unprivileged-tasks`.
+users.
 
 .. important::
 
-   Red Hat OpenShift users should not configure non-root containers.
+   Red Hat® OpenShift® users users should not configure non-root containers.
+
+To run containers as non-root users, you'll first need to set up your non-root user:
+
+-  Choose an existing non-root user or create a new one for running containers.
```

Comment:
There are two sets of users to consider right now:
- Determined users, stored in the determined database and configured with `det user ...`
- System users, or so-called "agent users" (horrible name, in retrospect), which are stored somewhere in the operating system and configured with operating system commands like `useradd` or `usermod`.

So "choose an existing non-root user" is ambiguous, because the reader doesn't know which kind of user you are asking about.

This should say "choose an existing *determined* user".

However, when you add "existing *non-root* user" into the mix, then it is additionally confusing because determined has "admin" and "non-admin" users, not "root" and "non-root" users.  So if you don't like my original phrasing of "who should run as non-root", I don't blame you, but calling a determined user "non-root" isn't very clear.

I'm not sure how to specify it other than:

> Pick a determined user, it doesn't matter who, but probably one you haven't already done `det user link-with-agent-user` on, and probably one you intend to link with an agent user eventually.  If no such determined user exists, create a test user you can disable afterwards".